### PR TITLE
everything: Remove filename from @file Doxygen command

### PR DESCRIPTION
**Pull request by gebart**
Doxygen can figure out the filename by itself if it is not given as an argument to the `@file` command.

`git grep -l -E '@file\s' | xargs sed -i -e 's/@file\s.*$/@file/'` was used to remove the filename from the `@file` blocks.

### CODE REVIEW

File: boards/mbed_lpc1768/system.c [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**************************************************************************//**
- * @file     boards_init.c
```
Comment (gebart) [inline boards/mbed_lpc1768/system.c]: The old filename argument did not even match the real file name here.

**Comment by gebart** [comment]
I've done a brief check of the results (including the HTML output from Doxygen) before posting this PR and it seems fine.

File: boards/mbed_lpc1768/system.c (hunk 1 above)
Comment (OlegHahm) [inline boards/mbed_lpc1768/system.c]: I wonder who commits such a nonsense. :innocent:

**Comment by OlegHahm** [comment]
ACK - please rebase.

**Comment by LudwigOrtmann** [comment]
"Showing 342 files with 300 additions and 300 deletions." << this seems odd ;)

File: cpu/k60/include/MK60D10.h [hunk 2]
```diff
@@ -73,7 +73,7 @@
 */
 
 /*!
- * @file MK60D10.h
+ * @file
```
Comment (LudwigOrtmann) [inline cpu/k60/include/MK60D10.h]: I keep forgetting the details - didn't we want to leave these CMSIS files unchanged (except for trailing whitespace)?

**Comment by LudwigOrtmann** [comment]
Other then the CMSIS files everything seems in order.

**Comment by gebart** [comment]
Regarding the 300, that's a limit in github's viewer, it only counts and shows the first 300 files changed. There's a warning message at the top of the page when you navigate to the files changed tab. 

I can revert the vendor supplied headers in cpu if you like?

**Comment by gebart** [comment]
rebased (re-ran the regex on latest master) and excluded `cpu/cortex-m?_common/include`, `cpu/k60/include`, `cpu/stm32*/include` (vendor supplied headers)

**Comment by gebart** [comment]
Travis is green, ok to merge?

**Comment by LudwigOrtmann** [comment]
Yo

**Pull request merged**